packages/hint-doctype/README.md

```diff
@@ -0,0 +1,72 @@
+# Check that the Doctype tag of the document is valid
+
+This hint checks if there is a valid Doctype in the document.
+
+## Why is this important?
+
+A valid doctype will give the browser the correct information
```

Comment:
I like MDN's explanation. Maybe we can just put it as a comment and link the source:

> In HTML, the doctype is the required "`<!DOCTYPE html>`" preamble found at the top of all documents. Its sole purpose is to prevent a browser from switching into so-called “quirks mode” when rendering a document; that is, the "`<!DOCTYPE html>`" doctype ensures that the browser makes a best-effort attempt at following the relevant specifications, rather than using a different rendering mode that is incompatible with some specifications.

https://developer.mozilla.org/en-US/docs/Glossary/Doctype

Reply:
Can you confirm that this is right ?